src/Stratis.Bitcoin.IntegrationTests/Miners/ProofOfStakeTests.cs

```diff
@@ -0,0 +1,46 @@
+﻿using Stratis.Bitcoin.Consensus;
+using Stratis.Bitcoin.IntegrationTests.Common;
+using Stratis.Bitcoin.IntegrationTests.Common.EnvironmentMockUpHelpers;
+using Stratis.Bitcoin.Tests.Common;
+using Xunit;
+
+namespace Stratis.Bitcoin.IntegrationTests.Miners
+{
+    public class ProofOfStakeTests
+    {
+        [Fact]
+        public void MiningAndPropagatingPOS_MineBlockCheckPeerHasNewBlock()
+        {
+            using (NodeBuilder nodeBuilder = NodeBuilder.Create(this))
+            {
+                CoreNode node = nodeBuilder.CreateStratisPosNode(KnownNetworks.StratisRegTest).NotInIBD().WithWallet();
```

Comment:
@thefazzer here you need to pass in a new instance of StratisRegTest otherwise, when you update the `LastPowBlock` it effects all other instances, which will affect tests that use that instance.

So pass in `new StratisRegTest` here and everywhere else in these tests.